packages/ckeditor5-image/src/imageresize.js

```diff
@@ -77,18 +78,18 @@ export default class ImageResize extends Plugin {
  *					resizeUnit: "%",
  *					resizeOptions: [ {
  *						name: 'imageResize:original',
- *						label: 'Original size',
  *						value: null
+ *						// you should add `icon` property if you're configuring the standalone buttons.
```

Comment:
Let's move these comments to the second part of the doc. They don't make sense if they configure a drop-down and they bring only noise.